lib/src/draw/draw_line.dart

```diff
@@ -1,5 +1,6 @@
 import 'dart:math';
 
+import '../../image.dart';

```

Comment:
The export file shouldn't be included here. You should import specific files.

Reply:
fixed